plugins/hls-brittany-plugin/hls-brittany-plugin.cabal

```diff
@@ -11,12 +12,13 @@ category:      Development
 build-type:    Simple
 extra-source-files:
   LICENSE
+  test/testdata/**/*.hs
 
 library
   exposed-modules:  Ide.Plugin.Brittany
   hs-source-dirs:   src
-  build-depends:    base
-               ,    brittany      >= 0.13.1.0
+  build-depends:    base             >=4.12 && <5
```

Comment:
Why not `base ^>= 4.12`?

Reply:
I copied it from the main package so I would keep it for consistency, this way we will accept base > 4.13 && < 5, not sure if it is a good thing though.
But I would change it in another pr, if you don't mind.

Reply:
Sure, the plan sounds good to me